_posts/2019-03-26-bazel-0.24.md

```diff
@@ -0,0 +1,74 @@
+
+This version of Bazel has been regression-tested against [over 50 open-source projects](https://buildkite.com/bazel/bazel-at-head-plus-downstream/builds/867). Want to get your open-source project on Bazel's CI to ensure that we catch any regressions or have suggestions what we should test against? Reach out to bazel-dev@googlegroups.com and we can set you up!
+
+# Incompatible changes
+The following incompatible flags are now enabled by default, and will be removed in a near-term release. Before updating to Bazel 0.24, you may first check if your codebase is compatible either by running [`bazelisk --migrate`](https://github.com/philwo/bazelisk) or by building your code with Bazel
+0.23 and the following flags:
+
+   - [`--incompatible_disable_runtimes_filegroups`](https://github.com/bazelbuild/bazel/issues/6942)
+   - [`--incompatible_linkopts_in_user_link_flags`](https://github.com/bazelbuild/bazel/issues/6826)
+   - [`--incompatible_dont_emit_static_libgcc`](https://github.com/bazelbuild/bazel/issues/6825)
+   - [`--incompatible_disallow_filetype`](https://github.com/bazelbuild/bazel/issues/5831)
+   - [`--incompatible_disallow_dict_plus`](https://github.com/bazelbuild/bazel/issues/6461)
+   - [`--incompatible_disallow_old_style_args_add`](https://github.com/bazelbuild/bazel/issues/5822)
+   - [`--incompatible_java_coverage`](https://github.com/bazelbuild/bazel/issues/7425)
+
+The following new incompatible flags have been added in this release. Projects are expected to begin migration:
+   - [`--incompatible_list_based_execution_strategy_selection`](https://github.com/bazelbuild/bazel/issues/7480): Migration to a new style of specifying  execution strategy selection and fallback behavior.
+   - [`--incompatible_windows_style_arg_escaping`](https://github.com/bazelbuild/bazel/issues/7454): Enables correct subprocess argument escaping on Windows. (No-op on other platforms.)
+   - [`--incompatible_windows_escape_jvm_flags`](https://github.com/bazelbuild/bazel/issues/7486): Enables correct `java_binary.jvm_flags` and `java_test.jvm_flags` tokenization and escaping on Windows. (No-op on other platforms.)
+   - [`--incompatible_disable_objc_library_resources`](https://github.com/bazelbuild/bazel/issues/7594): Disables resource attributes in `objc_library`. Please migrate these attributes to `data` instead.
+   - [`--incompatible_disallow_native_in_build_file`](https://github.com/bazelbuild/bazel/issues/7513): Enable migration while removing the use of the `native` module in BUILD files.
+   - [`--incompatible_use_toolchain_providers_in_java_common`](https://github.com/bazelbuild/bazel/issues/7186): Cause `java_common` APIs to require `JavaToolchainInfo` and `JavaRuntimeInfo` providers instead of configured targets.
+
+[`--incompatible_py3_is_default`](https://github.com/bazelbuild/bazel/issues/7359): Makes targets build for Python 3 unless otherwise specified.
```

Comment:
add `-` for the list (also the next item)

Reply:
Done, thanks for catching that.